packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -326,7 +330,10 @@ type BaseInputCommonProps = FormInputLabelProps &
      * @default true
      **/
     isTableInputCell?: boolean;
-
+    /**
+     * Callback to be invoked when the backdrop is clicked
+     */
+    onInputWrapperClick?: (event: React.MouseEvent<HTMLInputElement>) => void;
```

Comment:
Do we need this prop anymore? The trailingInteractionElement will handle the clicking and input already has onClick for rest of the part right?

Reply:
onTrailingInteractionElementClick will only work within the wrapper. If even the TrailingInteractionElement wrapper does not occupy the full space, an onInputWrapper click is still required.
![image](https://github.com/user-attachments/assets/997c0e20-da34-44b9-9158-db1ca401e882)

Reply:
update: we decided to go with 
![image](https://github.com/user-attachments/assets/325f1759-0b4c-4785-b36e-a1f29e5d0837)